### DISCUSSION TITLE
ESP32C3 and Tasmota 13.2.0

### DISCUSSION THREAD

**Discussion by ghost** [arendst/Tasmota#20986] [Q&A] [answered]
My ESP32C3 runs Tasmota 13.2.0 and "script". I read my SmartMeter and also use GoogleChart. Then suddenly there is a restart and "script" is not activated. The "script" runs for about 16-20 hours and then the restart. The variables will not be updated until the script is activated again. How can I prevent the reboot or enable "script" automatically after a reboot?

**Comment by sfromis** [comment, marked as answer]
You already got an answer on Discord, no reason to repeat the question.
https://discord.com/channels/479389167382691863/479389167382691865/1219640204588224682

**Reply by ghost** [reply]
Ok,

**Comment by HofeBY** [comment]
@Behmer1961
Your Name sound´s like you came from Germany
Well, I think to know that in TM13.2.x.x reboots have bin a problem.
Maybe you want to try my compile version on site
[github.com/HofeBY/Tasmota-Firmware](url)
read "Liesmich-zuerst.txt" what I have done to compile my own TM development version.
compare my "user_config_override.h" with your´s.
there is also an untestet "tasmota32c3.bin" . If you try out my version, let me know if it work´s well, or what has to be changed.

Oberfränkische Grüße
HofeBY
